### ISSUE TITLE
extension doesn't work with latest version of vscode

### ISSUE THREAD

**Issue by hellmoss** [rsKliPPy/amxxpawn-language#19]
https://forums.alliedmods.net/showpost.php?p=2753638&postcount=167

> The latest version of this extension doesn't work with the latest version of VS Code. 
> Because when you click on Compile Plugin or Compile Plugin Locally, it will always give an error "command amxxpawn.compile not found".
> I searched through stackoverflow and found that you need to activate the commands on "activation events" section in the "package.json" file and then recompile it.
> After recompiling, it works fine as it is expected.
> Klippy please update the extension by doing the following fixes, else if no one is going to do these, then I might publish the fixed version in the market place.
>

**Comment by fysiks1** [comment]
Would you like to provide any details?  It's working for me just fine for me and VS Code just updated 5 days ago (v1.59.1).

- What version of VS Code are you running?
- What version of the extension are you using?
- Are you on Windows, Linux, or Mac?

**Comment by hellmoss** [comment]
That post isn't actually mine but I'm experiencing the same issue.

I'm using latest VS Code 1.59.1 (5 days ago), latest version of extension; tried reinstalling it. Using Windows 10 x64 Pro v2004, 19041.1165. Will try reinstalling VS Code and see if that helps...

EDIT: reinstalling had no effect

**Comment by fysiks1** [comment]
Ok, so it looks like the only difference between us is the OS.  Unfortunately, I'm unable to test on a Windows computer.  Abhinash over on the alliedmods forums claims to have the same issue and also claims to have a fix but it's unclear what that fix is.  Hopefully he'll respond to my inquire about OS to see if it correlates to OS only.

**Comment by hellmoss** [comment]
Possibly solved: https://forums.alliedmods.net/showthread.php?t=300661&page=19


TL;DR: enable Trusted Workspace
